examples/verify-quorum/host/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2021"
 sp1-cc-host-executor = { path = "../../../crates/host-executor" }
 sp1-cc-client-executor = { path = "../../../crates/client-executor" }
 
-alloy-primitives.workspace = true
+alloy-primitives = { workspace = true, features = ["rand"] }
```

Comment:
what's the rand feature for?

Reply:
It's for B256::random_with()
(It wasn't feature gated previously)